doc/user-guide/schemas.rst

```diff
@@ -11,3 +11,36 @@ Schemas
         * Near-real-time "prompt" ConsDB replicates a subset of the USDF version
         * Data Release ConsDB is a snapshot of a subset of the USDF version with data pertaining to the exposures/visits in the DR
 * Schema browser
+
+* Versioning
+
+https://rubin-obs.slack.com/archives/C07QJMQ7L4A/p1730482605167509
+
+- schemas are using semantic versioning
+- should be consistent across all schemas, not just ConsDB
+
+    major: backward incompatible changes to the database objects (adding a table, deleting a column)
+
+        - except adding a table is not backwards incompatible
+
+    minor: backward compatible changes to the database objects (adding a column)
+    patch: updates or additions to semantics/metdata (units, UCDs, etc.)
+
+        - changing units can create incompatibilities
+
+    And we should say what should happen in the case of changes to primary/foreign keys.
+    Semantic neutrality: becoming non-primary is unique and anything becoming primary was already unique
+
+    - or there can be ones that are not neutral.
+
+    Think about the utility of these versions in terms of interaction with the ConsDB APIs, migrations, etc.
+
+    Do sdm_schemas versions appear in the db?
+
+    Currently the schemas are tagged and versioned as a set, at least w.r.t. the Science Platform.
+    So once ConsDB is available in TAP, it should be part of that set.
+
+    What do users see, how does TAP play into this, do the schema need this type of micro versioning?
```

Comment:
Unless the procedures change, ConsDB schemas will be deployed to the Science Platform as part of an sdm_schemas release. This version will _not_ be visible to users. This will determine what tables and columns are visible in TAP. We can cut new releases of sdm_schemas to incorporate changes to ConsDB as needed. (The sdm_schemas semver releases used on the Science Platform are made ad hoc/as needed and not on a fixed schedule.)

We will have the capability to deploy different releases of sdm_schemas to different environments if this is required (`usdf-rsp`, `usdf-rsp-dev`, etc.). But we do not currently have the capability to deploy different versions of specific schemas individually. They are versioned as a set.